### PR TITLE
Allow running heat kuttl test in any namespace

### PR DESCRIPTION
**Pull request by vhariria**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1170,9 +1171,12 @@ ironic_kuttl_crc: crc_storage ironic_kuttl
 
 .PHONY: heat_kuttl_run
 heat_kuttl_run: ## runs kuttl tests for the heat operator, assumes that everything needed for running the test was deployed beforehand.
-	kubectl-kuttl test --config ${HEAT_KUTTL_CONF} ${HEAT_KUTTL_DIR}
+	kubectl-kuttl test --config ${HEAT_KUTTL_CONF} ${HEAT_KUTTL_DIR} --namespace ${NAMESPACE}
 
 .PHONY: heat_kuttl
+	heat_kuttl: export NAMESPACE = ${HEAT_KUTTL_NAMESPACE}
```
[the remark this answers is not in the document]
Comment (vhariria) [reply Makefile]: Thanks for your feedback.

**Review by cescgina — COMMENTED** [review]
thanks, lgtm

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**